src/zope/password/password.py

```diff
@@ -450,13 +451,119 @@ def checkPassword(self, encoded_password, password):
         return encoded_password == self.encodePassword(password)[5:]
 
     def match(self, encoded_password):
-        if isinstance(encoded_password, unicode):
+        if isinstance(encoded_password, text_type):
             encoded_password = encoded_password.encode('ascii')
         return (
             encoded_password.startswith(b'{SHA}') or
             encoded_password.startswith(b'{SHA1}'))
 
 
+if bcrypt is not None:
```

Comment:
Making the `BCrypytPasswordManager` always be defined means that the doc-tests fail when bcrypt is not installed.
Would it be ok remove them since there are unittests?